### DISCUSSION TITLE
Missing EXIT command

### DISCUSSION THREAD

**Discussion by ChrisIsTounet** [Hofknecht/SystemTrayMenu#515] [Ideas]
at first great job ! your menu is amazing !

unfortunately there is no exit command !

i have checked "Check for update" and got a message with a new version available, so i downloaded it but trying to overwrite the old one
i got a system error : file used by another program.

i had to open task explorer and end the SystemTrayMenu task manually before updading !

it will be very helpfull to have a  exit button like the reload button at the bottom !

thanks.

**Comment by topeterk** [comment]
Not sure if I understood your point as when you right click the tray icon, you should be able to close the application.
![image](https://github.com/Hofknecht/SystemTrayMenu/assets/4932525/657183e3-bfae-4e7b-905e-3e61b7a119a2)
Is this what you were looking for?

**Comment by ChrisIsTounet** [comment]
sorry i am on W11 and i haven't even realized that there where a menu on tray icon, because it's hidden by default !
i allways start the menu by using Ctrl+Win

so this is fine thanks you very much for your help !

i close the thread please accept my apologies.

**Comment by ChrisIsTounet** [comment]
missing tray icon menu

**Comment by Hofknecht** [comment]
@ChrisIsTounet 
do you mean the tray icon is missing? 
maybe you see it if you open the hidden tray icon via click the arrow in taskbar?
![image](https://github.com/Hofknecht/SystemTrayMenu/assets/52528841/65b48f59-b60a-4d6d-9469-7d7fff73de68)

you can also exit via right click taskbar icon and "click close all windows"
![image](https://github.com/Hofknecht/SystemTrayMenu/assets/52528841/327343bf-b6ab-4c8c-962c-622da63ed95b)

you can check if something is in the logfile when you start, find the logfile in this path:
C:\Users\YourUser\AppData\Roaming\SystemTrayMenu

Do you use the portable version or the windows store version? Is there any difference if you use the other version?

**Reply by ChrisIsTounet** [reply]
you're right i miss to look for hiden icons, and also disable taskbar icon in the parameters so i miss exit in menu

hopefully thanks to you, now i know what to do to exit !
